### PR TITLE
Add logging implementation to Java sdk tests

### PR DESCRIPTION
**Pull request by boydjohnson**
This cleans up some warnings that are printed during `mvn package`.

### CODE REVIEW

File: sdk/java/src/test/java/sawtooth/sdk/client/SigningTest.java [hunk 1]
```diff
@@ -19,7 +19,6 @@
 import java.io.FileNotFoundException;
 import java.io.IOException;
 import java.lang.String;
-import java.math.BigInteger;
```
Comment (peterschwarz) [inline sdk/java/src/test/java/sawtooth/sdk/client/SigningTest.java]: Is this change relevant of for this PR, or should it at least be in a separate commit?

Comment (boydjohnson) [reply sdk/java/src/test/java/sawtooth/sdk/client/SigningTest.java]: It could be in a separate commit. I removed 2 unneeded imports.

Comment (boydjohnson) [reply sdk/java/src/test/java/sawtooth/sdk/client/SigningTest.java]: I think I am going to drop the removal of the imports from the PR.
